### ISSUE TITLE
WHiCH (Western Himalaya Canopy Height) Map 2020, Pakistan

### ISSUE THREAD

**Issue by adeelpu** [samapriya/awesome-gee-community-datasets#410]
### Contact Details

adeel.geog@pu.edu.pk

### Dataset description

The western Himalayas region is a key biodiversity hotspot and a habitat for several species. This region needs an accurate spatial assessment of tree canopy height for improved estimates of aboveground biomass, carbon sequestration, and related forest ecosystem services. In this work, we used a deep convolution neural networks model, U-Net, for estimating canopy heights using Global Ecosystem Dynamics Investigation Mission (GEDI) with multi-band Sentinel-2 datasets. We produced a 10 m canopy height map of the study region for 2020. Overall, we achieved a Root Mean Square Error (RMSE) of 7.52 m and a Mean Absolute Error (MAE) of 5.71 m on test set, outperforming existing global canopy height models in this topographically challenging region.

Original research article:

Ahmad, A., Sastry, S., Dhakal, A., Khanal, S., Levering, A., Gilani, H. & Jacobs, N. (2025). Canopy Height Mapping in the Western Himalayas, Pakistan: A Deep Learning Approach using GEDI and Sentinel-2 Fusion. International Journal of Applied Earth Observation and Geoinformation. https://doi.org/10.1016/j.jag.2025.105030

### Earth Engine Snippet if dataset already in GEE

for example
```js
var WhiCH_Map = ee.Image("users/adeelgeog/WHiCH_Map")
```
Sample Code: Add a sample code maybe just adding your datasets in the code editor

### Enter license information

CC-BY-4.0

### Keywords

Western Himalayas, Canopy height, U-net CNN, GEDI, Sentinel-2, Stratified training

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct
